test/functional/eval/let_spec.lua

```diff
@@ -59,10 +59,6 @@ describe(':let', function()
   end)
 
   it("multibyte env var to child process #8398 #9267",  function()
-    if (not helpers.iswin()) and helpers.isCI() then
-      -- Fails on non-Windows CI. Buffering/timing issue?
-      pending('fails on unix CI', function() end)
-    end
```

Comment:
The problem on non-Windows apparently might just have been that it was writing to stderr there.